VideoLocker/res/values/styles.xml

```diff
@@ -299,13 +298,13 @@
 
     <!-- Spinner with the popup style -->
     <style name="edX.Widget.ListPopupWindow" android:parent="@style/Widget.AppCompat.ListPopupWindow">
-        <item name="android:popupBackground">?android:attr/popupBackground</item>
+        <item name="android:popupBackground">@drawable/popup_menu</item>
```

Comment:
I guess it can be “popup_menu_background”; the “bg" abbreviation I’d only use as a prefix.
> On 21-Oct-2015, at 2:51 PM, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/values/styles.xml <https://github.com/edx/edx-app-android/pull/400#discussion_r42603642>:
> 
> > @@ -299,13 +298,13 @@
> >  
> >      <!-- Spinner with the popup style -->
> >      <style name="edX.Widget.ListPopupWindow" android:parent="@style/Widget.AppCompat.ListPopupWindow">
> > -        <item name="android:popupBackground">?android:attr/popupBackground</item>
> > +        <item name="android:popupBackground">@drawable/popup_menu</item>
> the drawable is only being used as a background, why not name it something like popup_menu_bg
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/400/files#r42603642>.
>